guides/upstream_filename_to_link.json

```diff
@@ -0,0 +1,19 @@
+{
+  "build/Administering_Project/index-foreman-el.html": "administering_red_hat_satellite",
```

Comment:
I'd say `index-katello.html` is much closer to the downstream builds. Why did you choose `index-foreman-el.html`?